util/util.go

```diff
@@ -174,6 +176,16 @@ func GetLoggingWriter(cfg *config.Config) (io.Writer, error) {
 			MaxAge:     28,   // days
 			Compress:   true, // disabled by default
 		}
+		// rotate log file on SIGHUP
```

Comment:
This is awesome! But this needs to be in `cmd/garm/main.go`. Any signal handlers need to be handled there.

After we call `GetLoggingWriter()` we can spawn the bellow go routine and wait for a `SIGHUP` signal.  If we have a lumberjack logger, we can call `Rotate()` like you're already doing.